.travis.yml

```diff
@@ -28,9 +28,7 @@ matrix:
 
 before_install:
     - composer self-update
-    - if [ "$SYMFONY_VERSION" != "" ]; then
```

Comment:
Travis removes the line return character so it is parsed as `if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;`.

I would rather remove `.travis.yml` from Insight check.